qa/rpc-tests/pegging.py

```diff
@@ -254,6 +259,29 @@ def write_bitcoin_conf(datadir, rpcport, rpcpass=None, p2p_port=None, connect_po
         if "confirmations" not in tx or tx["confirmations"] == 0:
             raise Exception("Peg-in confirmation has failed.")
 
+    print ("Now test failure to validate peg-ins based on intermittant bitcoind rpc failure")
+    bitcoin2.stop()
+    txid = bitcoin.sendtoaddress(addrs["mainchain_address"], 1)
+    bitcoin.generate(12)
+    proof = bitcoin.gettxoutproof([txid])
+    raw = bitcoin.getrawtransaction(txid)
+    stuck_peg = sidechain.claimpegin(raw, proof)
+    sidechain.generate(1)
+    print("Waiting to ensure block is being rejected by sidechain2")
+    time.sleep(5)
+
```

Comment:
Perhaps call sync_all(sidechain, sidechain2) here and before the next assert?

Reply:
not here, since they don't sync, but yes subsequent makes sense!

Reply:
actually no.... we are waiting for bitcoind to be up. We could spin-wait trying to connect at least.

Reply:
3rd thoght, you're right, will fix